src/utils.ts

```diff
@@ -26,6 +27,18 @@ export function getCheckRunPayloadWithScanResult(trivyStatus: number, dockleStat
   return checkRunPayload;
 }
 
+export function getScanReport(): string {
+  const scanReportPath = `${fileHelper.getContainerScanDirectory()}/scanReport.json`;
+  const trivyOutput = trivyHelper.getTrivyFilteredOutput();
+  const dockleOutput = dockleHelper.getDockleFilteredOutput();
```

Comment:
Check Dockle return code. If it is not equal to DOCKLE_EXIT_CODE, set this to an empty array?

Reply:
Do you mean checking add-cis-enable input? If DOCKLE_EXIT_CODE is 0, i.e. no vulnerabilities are detected, then it's already set to an empty array. However, if add-cis-enable is set to false, it would give an error as it's trying to read dockleoutput.json which does not exist. I'll fix that.

Reply:
I think both (`add-cis-enable` and `statusCode === DOCKLE_EXIT_CODE`) should be checked.

We should try to read the output json only if the status code is DOCKLE_EXIT_CODE, that is Dockle says some violations are found. Otherwise (i.e., no failures and error case) we know that it should be an empty array.

Reply:
Added the check for both dockle and trivy